docs/src/index.md

```diff
@@ -43,35 +43,30 @@ In order to add a new backend,
 2. edit `src/requires.jl` to define how to produce the plot from the data:
     ```julia
     @require SomeNewPlot = "91a5bcdd-55d7-5caf-9e0b-520d859cae80" begin
-      function performance_profile(::SomeNewPlotBackend,
-                                  T :: Matrix{Float64},
-                                  labels :: Vector{S}=String[];
-                                  logscale :: Bool = true,
-                                  title :: AbstractString = "",
-                                  sampletol :: Float64 = 0.0,
-                                  drawtol :: Float64 = 0.0,
-                                  kwargs...) where S <: AbstractString
-        ns = size(T, 2)
-        xlabel, ylabel, labels = performance_profile_axis_labels(labels, ns, logscale; kwargs...)
-        (x_plot, y_plot, max_ratio) = performance_profile_data(T, logscale=logscale, sampletol=sampletol, drawtol=drawtol)
-        kwargs = Dict{Symbol,Any}(kwargs)
+      function performance_profile_plot(::SomeNewPlotBackend,
+                                        x_plot,
+                                        y_plot,
+                                        max_ratio,
+                                        xlabel,
+                                        ylabel,
+                                        labels,
+                                        title,
+                                        logscale;
+                                        kwargs...)
```

Comment:
```suggestion
      function performance_profile_plot(
        ::SomeNewPlotBackend,
        x_plot,
        y_plot,
        max_ratio,
        xlabel,
        ylabel,
        labels,
        title,
        logscale;
        kwargs...
      )
```

Reply:
I've been slowly changing to this style, suggested by in https://github.com/invenia/BlueStyle. It's easier to read and to indent, so I like it. Let me know if you agree, in which case I can make the modifications in this PR

Reply:
I’d rather keep PRs focused. Stylistic changes should go into another PR. What I’d really like is a linter that runs in an action and either fails when there are formatting errors, or modifies the PR to suggest changes.

Reply:
You're right. It might be possible to create that action (https://github.com/invenia/BlueStyle/issues/7) but I don't know how easy it would be.